vsintegration/src/vs/FsPkgs/FSharp.VS.FSI/fsiCommands.vsct

```diff
@@ -119,7 +119,6 @@
       <!-- The following places a button on the F# Editor Context Menu -->
       <!-- In Dev11, shell now has 
       <Button guid ="guidVSStd11" id ="cmdidExecuteSelectionInInteractive" priority ="0x100" type ="Button">
-      <Button guid ="guidVSStd11" id ="cmdidExecuteLineInInteractive" priority ="0x105" type ="Button">
```

Comment:
(sorry GH won't let me leave comment for a line in this file that's not part of the diff)  There are more usages of "send line" command in this file that should be removed.  Just scan for "line" and you will find them.